### ISSUE TITLE
Error when trying to compile

### ISSUE THREAD

**Issue by jonasw8** [noah-nuebling/mac-gamepad-driver#1]
Hello! I have the xbox 360 wireless joystick and would like to help with feedback, but I am receiving an error message when trying to compile. Attached is a screenshot of the error
![Captura de Tela 2023-09-19 às 13 49 16](https://github.com/noah-nuebling/mac-gamepad-driver/assets/53247355/322bc451-98e6-46fd-b0d2-dcc0972c66f5)

**Comment by noah-nuebling** [comment]
You gotta turn off automatic signing I think. Use Ad-Hoc signing I stead which might be the same as no signing at all but not sure

**Comment by jonasw8** [comment]
I tried ad-hoc, but I received this error. Would it be possible for you to compile it and send it to me? I would like to test this kext
![Captura de Tela 2023-09-19 às 22 06 28](https://github.com/noah-nuebling/mac-gamepad-driver/assets/53247355/76c25483-7108-4092-b3e2-f1c06aa5b74c)

**Comment by noah-nuebling** [comment]
I don't think it makes sense to send you my compiled version since the source code will only connect to my specific controller model which is some weird 3rd party offbrand one. The source code contains the vendorID and productID for my specific controller and macOS will only load the driver if the vendorID and productID match. So you'll have to add the vendorID and productID of your controller to the project in order for macOS to load the driver when you connect your controller.

Maybe you can get around the issues you're having by following the solution at the bottom of this document: https://github.com/noah-nuebling/mac-gamepad-driver/blob/main/gamepad-driver-user/Notes.md (Turn off System Integrity Protection, Turn on developer mode of the systemextensionsctl command line tool, and maybe some more stuff.)

**Comment by jonasw8** [comment]
My Product ID e Vendor ID in hexdecimal(extracted from system) - Joysctick: Xbox 360 Wireless
Vendor ID: 0x045e
Product ID: 0x0719

I have already disabled SIP and used the systemextensionsctl command, but the error persists

**Comment by noah-nuebling** [comment]
Ohhh, this doesn't support wireless, only USB. This just ports the USB driver from the original 360Controller project. For wireless they implemented a totally different driver afaik.

Sorry to waste your time.

**Comment by jonasw8** [comment]
You didn't waste my time. I'm glad someone is rewriting kext for newer systems. If you ever manage to port the driver for the wireless version, I can be a tester and provide all the necessary feedback.
